setup/fedora.md

```diff
@@ -0,0 +1,193 @@
+# Development Environment Setup on Fedora 28
+
+This is a short guide on how to set up the compiler and interpreter for prototype Xalgorithm Rules on **Fedora 28**. A similar process could be followed on most GNU/Linux distributions or within the Windows Subsystem for Linux. Current interpreter builds (July 2018) use JSON representations of tables, rules, package information and documents. Refer to the [project documentation](https://github.com/xalgorithms/general-documentation) for more information about the working implementation.
+
+Configuring these repositories will prepare all prerequisites for contributing to the project by improving the rule interpreter.
+
+## What is an Xalgorithm Rule?
```

Comment:
Remove this section and change it to refer to definitions that we have elsewhere in documentation. We don't exactly have this description in a single document. If there's something missing that you've documented here, open an issue to extend the particular document.

Reply:
Sure. This was originally written to be a blog post, I'll cut some of the exposition out.